HACKING.md

```diff
@@ -121,7 +128,7 @@ cmake --build .
 ctest
 ```
 
-In some cases, you may want to specify your compiler, especially if the default compiler on your system is too old. You may proceed as follows:
+In some cases, you may want to specify your compiler, especially if the default compiler on your system is too old.  You need to tell cmake which compiler you wish to use by setting the CC and CXX variables. Under bash, you can do so with commands such as `export CC=gcc-7` and `export CXX=g++-7`. You can also do it as part of the `cmake` command: `cmake .. -DCMAKE_CXX_COMPILER=g++`.  You may proceed as follows:
```

Comment:
For context, the -DCMAKE_CXX_COMPILER is something you'd only really do if you wanted to *change* from one compiler to another ... cmake is willing to set up its initial choices based on variables, but it won't change them further after that without -D. This is fine, though.

Reply:
By which I mean, this wording is good, not "you should add my pedantry into your wording" :)